wolfhsm/wh_common.h

```diff
@@ -24,54 +24,13 @@
 #ifndef WOLFHSM_WH_COMMON_H_
 #define WOLFHSM_WH_COMMON_H_
 
+#include <stdint.h>
+
 /* Pick up compile-time configuration */
 #include "wolfhsm/wh_settings.h"
```

Comment:
```suggestion
/* Pick up compile-time configuration */
#include "wolfhsm/wh_settings.h"

#include <stdint.h>
```
Other library files have wh_settings.h before anything else, just in case.